core/templates/domain/story/story-node.model.spec.ts

```diff
@@ -318,4 +318,13 @@ describe('Story node model', () => {
 
     expect(_sampleStoryNode.getThumbnailBgColor()).toEqual('#fff');
   });
+
+  it('should return false when nodeId is not a string', () => {
```

Comment:
I don't see the `action` part in the test message `Story node model should return false when nodeId is not a string` i.e, what's the action to return `false`?  Also, don't use the private method for testing.

Reply:
@nithusha21 are we expecting to add frontend test using `private` methods?